src/lib/chromex/chrome_content_setting.cljs

```diff
@@ -0,0 +1,39 @@
+(ns chromex.chrome-content-setting
+  (:refer-clojure :exclude [get set remove])
+  (:require [chromex.support :refer [get-hook]]
+            [oops.core :refer [ocall ocall!]]))
+
+(defprotocol IChromeContentSetting
```

Comment:
I had to put this here instead of protocols.cljs because there are collisions on the symbols (get,  set, etc) with StorageArea.

Reply:
Now I understand. I tackled it in #17.